pmd-cli/src/main/java/net/sourceforge/pmd/cli/commands/internal/PmdCommand.java

```diff
@@ -310,7 +326,15 @@ protected ExecutionResult execute() {
             TimeTracker.startGlobalTracking();
         }
 
-        final PMDConfiguration configuration = toConfiguration();
+        PMDConfiguration configuration = null;
+        try {
+            configuration = toConfiguration();
+        } catch (IllegalArgumentException e) {
+            System.err.println("Cannot start analysis: " + e);
+            LOG.debug(ExceptionUtils.getStackTrace(e));
```

Comment:
rather than doing this, let's catch the exception within `toConfiguration`, and wrap it in a `ParameterException`, the same way we do in case a failure setting up the auxclasspath. This will provide a consistent and better output.

Reply:
+1

Reply:
actually, this exception is never thrown on this code path. The paths we get via `--relativize-paths-with` have been already verified (in setRelativizePathsWith()). The IllegalArgumentException would only be thrown when PMD is used programmatically.

I've removed this special handling.